java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v2/impl/QueryMenuControllerImplV2.java

```diff
@@ -0,0 +1,16 @@
+package com.xxAMIDOxx.xxSTACKSxx.menu.api.v2.impl;
+
+import com.xxAMIDOxx.xxSTACKSxx.menu.api.v1.dto.response.MenuDTO;
+import com.xxAMIDOxx.xxSTACKSxx.menu.api.v2.QueryMenuControllerV2;
+import java.util.UUID;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class QueryMenuControllerImplV2 implements QueryMenuControllerV2 {
+
+  @Override
+  public ResponseEntity<MenuDTO> getMenu(UUID id, String correlationId) {
+    return null;
```

Comment:
Just duplicate the implementation of the v1 end point. Better to make it actually work than leave like that.